types/koa-react-view/index.d.ts

```diff
@@ -0,0 +1,24 @@
+// Type definitions for koa-react-view v3.0.0
+// Project: https://github.com/koajs/react-view
+// Definitions by: ephoton <https://github.com/ephoton>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module 'koa-react-view' {
```

Comment:
The CI failure is from declaring this as a module rather than declaring as external module